fe-nc-news-react/src/Components/Articles.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { useState } from "react";
+
+import ArticleList from "./ArticleList";
+
+function Articles() {
+  const [articles, setArticles] = useState();
+  return (
+    <div>
+      <ArticleList articles={articles} setArticles={setArticles} />
+    </div>
+  );
```

Comment:
There's no need for this Articles component if its only job is to render another component.